utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -299,9 +302,13 @@ def scan(self, image, scan_args):
             chroot = self._find_chroot_path(_tmp_mnt_dir)
 
             # Scan the chroot
-            sys.stdout.write(self.helper._scan(chroot, image, scan_args))
+            scan_result = self.helper._scan(chroot, image, scan_args)
+            print(scan_result.stdout)
+            print(scan_result.stderr, file=sys.stderr)
```

Comment:
Will it work with Python 2 ?

Reply:
Yes, there is the `from __future__ import print_function` at the top of the file.